NatureArea.py

```diff
@@ -55,6 +107,8 @@ def set_descriptions(self):
 
         National park format: "national park in Sweden".
         Reserve format: "nature reserve in <län>, Sweden".
+
+        :return: nothing
```

Comment:
Follow-up task to at least research whether the `return`s belong home there in the first place https://phabricator.wikimedia.org/T165936